docs/software/firmware/csv_format.md

```diff
@@ -0,0 +1,64 @@
+
+Headline  | Format | Range | Sample | Description |
+---       | --- | --- | --- | --- |
+`Date`    | TT.MM.YYYY | | 24.11.2020 | UTC, typically as received by the GPS module in that second. If there is no GPS module present, system time is used. If there was no reception of a time signal yet, this might be unix time (starting 1.1.1970) which can be at least used as offset between the csv lines.    
+`Time`      | HH.MM.SS | | 12:00:00 | UTC time, see also above
+`Latitude`  | Float |  | 9.123456 | Latitude as degrees
+`LatitudeDir` | Char | `N` /  `S` | N | Latitude direction as reported by the GPS module (GPGGA)
+`Longitude` | Float |  | 42.123456 | Longitude in degrees
+`LongitudeDir` | Char | `E` / `W` | W | Longitude direction as reported by the GPS module (GPGGA)
+`Altitude`  | Float | -9999.9 - 17999.9 | 480.12 | meters above mean sea level (GPGGA)
+`Course`    | Float | 0 - 359.9 | 42 | Course over ground in degrees (GPRMC)
+`Speed`     | Float | 0 - 359.9 | 42.0 | Speed over ground in km/h
+`HDOP` | Float | 00.0 - 99.9 | 2.3  | Relative accuracy of horizontal position (GPGGA)
+`Satellites` | Integer | 0-99 | 5 | Number of satellites in use (GPGGA)
+`BatteryLevel` | Float | 0-9.99 | 3.3 | Current battery level reading (~V)
+`Left`      | Integer | 0-999 | 150 | Left minimum measured distance in centimeters of this line, 999 for no measurement. 
+`Right`     | Integer | 0-999 | 150 | Right minimum measured distance in centimeters of this line, 999 for no measurement.
+`Confirmed` | Boolean | 0-1 | 1 | Measurement was confirmed overtaking by button press  
+`Marked`    | Boolean | 0-1 | 1 | Measurement was marked (not possible yet)
+`False`     | Boolean | 0-1 | 1 | Measurement was marked as false reading (not possible yet)
+`insidePrivacyArea`| Boolean | 0-1 | 1 | 
+`Factor`    | Integer |  | 58 | The factor used to calculate the time given in micro seconds (us) into centimeters (cm). Currently fix, might get adjusted by temperature some time later. |
+`Measurements` | Integer | 0-999 | 18 | Number of measurements taken in this line |
+            |      | | | | Now follows a series of #`Measurements` repetitions of #`DatasPerMeasurement` entries, `<n>` is always increased starting from 1 for the 1st measurement. Order is always the same, additional data might be added to the end, `DatasPerMeasurement` will be increased then.  |
+`Tms<n>`    | Integer | 0-1999 | 234 | Millisecond (ms) offset of measurement in this series (line) of measurements |
+`Lus<n>`    | Integer | 0-1000000 | 3456 | Microseconds (us) till the echo was received by the left sensor, divide by the `Factor` given above to get the distance in centimeters. -1 for no measurement taken. | 
+`Rus<n>`    | Integer | 0-1000000 | 3456 | Microseconds (us) till the echo was received by the right sensor, divide by the `Factor` given above to get the distance in centimeters. -1 for no measurement taken. |
```

Comment:
We should clarify whether the measurement is corrected for the  offset sensor <-> end-of-handlebar.
I assume it is not (which makes sense here). To receive a distance comparable to "Left" and "Right", we need not only to divide by Factor, but also subtract an offset. We should point out the latter, and give some hint where to get this information.

Reply:
Understand - I'll add the clarification. The offsets should be part of the "metadata" we send in the yet to be defined json header.